src/regex.js

```diff
@@ -39,6 +39,9 @@ SVG.regex = {
   // Test for image url
 , isImage:          /\.(jpg|jpeg|png|gif|svg)(\?[^=]+.*)?/i
 
+  // Test for url reference
+, isUrl:            /url\(([-\w:/]+(?:\.\w+)?)?#([-\w]+)\)/
```

Comment:
I like to be loose as possible by avoing the `.*` operator. I will go for `/(url\()?([-\w:/.]+)?#([-\w]+)/` which should match everything valid.